benchmarks/imagenet/resnet50/main.py

```diff
@@ -45,6 +47,8 @@
     "dino": {"model": dino.DINO, "transform": dino.transform},
     "simclr": {"model": simclr.SimCLR, "transform": simclr.transform},
     "swav": {"model": swav.SwAV, "transform": swav.transform},
+    "dcl": {"model": dcl.DCL, "transform": simclr.transform},
+    "dclw": {"model": dclw.DCLW, "transform": simclr.transform},
```

Comment:
```suggestion
    "dcl": {"model": dcl.DCL, "transform": dcl.transform},
    "dclw": {"model": dclw.DCLW, "transform": dclw.transform},
```

Could you also move them to the top of the dict? Then everything stays alphabetically sorted.